plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/sdk/AppEngineSDKClasspathContainer.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.sdk;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IClasspathContainer;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.JavaCore;
+
+import com.google.cloud.tools.appengine.cloudsdk.PathResolver;
+
+public final class AppEngineSDKClasspathContainer implements IClasspathContainer {
+
+  private static final String SDK_JAR = "/platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar";
+  public static final String CONTAINER_ID = "AppEngineSDK";
+
+  @Override
+  public IPath getPath() {
+    return new Path(AppEngineSDKClasspathContainer.CONTAINER_ID);
+  }
+
+  @Override
+  public int getKind() {
+    return IClasspathEntry.CPE_CONTAINER;
+  }
+
+  @Override
+  public String getDescription() {
+    return "App Engine SDKs";
+  }
+
+  @Override
+  public IClasspathEntry[] getClasspathEntries() {
+    java.nio.file.Path cloudSdkPath = PathResolver.INSTANCE.getCloudSdkPath();
+    if (cloudSdkPath != null) {
+      String appEngineToolsApiJar = cloudSdkPath + SDK_JAR;
```

Comment:
should use resolve() method here from java.nio.file.Path to better handle platform differences on windows.